src/test/end-to-end/pages/solicitorApplyCaveat/applyCaveat/page1.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const commonConfig = require('src/test/end-to-end/pages/common/commonConfig');
+
+module.exports = async function () {
+    const I = this;
+    await I.waitForElement('#solsCaveatEligibility');
+    await I.waitForNavigationToComplete(commonConfig.continueButton);
+};
```

Comment:
is this step actually doing anything other than wait? if not, then maybe the name of "applyCaveat" might be a little confusing, as it sounds like it's actioning something

Reply:
this page is the first page of applyCaveat event  where it shows some text for the user which is part of div tag with id solsCaveatEligibility. And once the solicitor reads the content he can click on continue to move further in the applying caveat process

Reply:
ah I see, so the `waitForNavigationToComplete()` step actually performs a click as well as wait, that's where I got thrown. Maybe a small improvement for a future PR but not in the scope of your changes 👍